config/packages/doctrine.yaml

```diff
@@ -13,7 +13,7 @@ doctrine:
                 commented: false
     orm:
         auto_generate_proxy_classes: true
-        naming_strategy: doctrine.orm.naming_strategy.underscore
+        naming_strategy: doctrine.orm.naming_strategy.underscore_number_aware
```

Comment:
Cannot create DIF, throws a doctrine error : column doesn't exist

Reply:
May need to run a migration.

Reply:
It did need a migration.